frontend/src/metabase/dashboard/components/DashCard/DashCard.tsx

```diff
@@ -116,6 +117,7 @@ function DashCardInner({
   onUpdateVisualizationSettings,
   onReplaceAllDashCardVisualizationSettings,
   downloadsEnabled,
+  shouldAutoScrollTo,
```

Comment:
nit: similar to "autoFocus", "autoScroll" might also work well as a variable name

Reply:
A variable named "shouldAutoScrollTo" feels, to me, like its value should be the thing that we should auto-scroll to

Reply:
Alternative: `shouldScrollToThisCardOnMount`

Reply:
Think about adding a comment to the interface explaining the prop if you give it a short name like `autoScroll`